components/dashboard/src/Login.tsx

```diff
@@ -126,6 +136,12 @@ export function Login() {
                         </div>
                     </div>
                 </div>
+                {emailTaken && (<div className="bg-gitpod-kumquat-light rounded-md p-3 text-gitpod-red text-sm mb-2">
```

Comment:
**nitpick**: Alerts like this can use a larger radius to differentiate them from inputs or other elements as we do in the garbage collection message in the workspaces lists.

Some thoughts on this:
1. This pattern will become more clear and consistent as its product design evolves.
2. Similarly, colors used for each alert type will also become more clear as design guidelines shape up.

```suggestion
                {emailTaken && (<div className="bg-gitpod-kumquat-light rounded-xl p-3 text-gitpod-red text-sm mb-2">
```

Reply:
Could we add some reuseable components for that, please? Each time I pick one example from somewhere else, it's the wrong one.

Reply:
Strong vote for YES!

This is common due to a) some design experimentations added on purpose in some pages and b) our MVC approach with helps us ship faster and unblock code reviews. The need to introduce a reusable component library will become more clear on the way as a) the product evolves and b) team members try to add or improve product features on more areas.

For this iteration, going with the `rounded-xl` class should suffice. WDYT?

Reply:
Sure! ☝🏻 updated the screenshots in the description above.